modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-6_{context}"]
+= Release notes for {pipelines-title} General Availability 1.6
+
+With this update, {pipelines-title} General Availability (GA) 1.6 is available on {product-title} 4.7, 4.8 and 4.9.
+
+[id="compatibility-support-matrix-1-6_{context}"]
+== Compatibility and support matrix
+
+Some features in this release are currently in Technology Preview. These experimental features are not intended for production use.
+
+link:https://access.redhat.com/support/offerings/techpreview[Technology Preview Features Support Scope]
+
+In the table, features are marked with the following statuses:
+
+- *TP*: _Technology Preview_
+
+- *GA*: _General Availability_
+
+Note the following scope of support on the Red Hat Customer Portal for these features:
+
+.Compatibility and support matrix
+[cols="1,1,1",options="header"]
+|===
+| Feature   | Version       | Support Status
+| {product-title} | 4.9     | GA
+| Pipelines Operator | 1.6  | GA
+| Pipelines | 0.28.x        | GA
+| CLI       | 0.21.x        | GA
+| Catalog   | 0.28          | GA
+| Triggers  | 0.16.x        | GA
+| Pipeline resources | -    | TP
+|===
+
+For questions and feedback, you can send an email to the product team at pipelines-interest@redhat.com.
+
+[id="new-features-1-6_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.6.
+
+=== CLI 0.21.0 release
```

Comment:
do all headings need an ID?

Reply:
Updated. Thanks. I have removed the versions but kept the categories that were presented to me in this source document: https://docs.jboss.org/display/RHDEVDOCS/OpenShift+Pipelines+1.6+documentation+plan#OpenShiftPipelines1.6documentationplan-Candidatesfor1.6RNs